CONTRIBUTING.md

```diff
@@ -0,0 +1,65 @@
+# Contributing to predicates-rs
+
+Thanks for wanting to contribute! There are many ways to contribute and we
+appreciate any level you're willing to do.
+
+## Feature Requests
+
+Need some new functionality to help?  You can let us know by opening an [issue][new issue]. It's helpful to look through [all
+issues][all issues] in case its already being talked about.
+
+## Bug Reports
+
+Please let us know about what problems you run into, whether in behavior or
+ergonomics of API.  You can do this by opening an [issue][new issue].. It's
+helpful to look through [all issues][all issues] in case its already being
+talked about.
+
+## Pull Requests
+
+Looking for an idea? Check our [issues][issues]. If it's look more open ended,
+it is probably best to post on the issue how you are thinking of resolving the
+issue so you can get feedback early in the process. We want you to be
+successful and it can be discouraging to find out a lot of re-work is needed.
+
+Already have an idea?  It might be good to first [create an issue][new issue]
+to propose it so we can make sure we are aligned and lower the risk of having
+to re-work some of it and the discouragement that goes along with that.
+
+### Process
+
+When you first post a PR, it might be good to clean up the commit history.  We
+recommend avoiding this during the PR to make it easier to review how feedback
+was handled. Once the commit is ready, we'll ask you to clean up the commit
+history.  Once you let us know this is done, we can move forward with merging!
+If you are uncomfortable with these parts of git, let us know and we can help.
+
+We ask that all new files have the copyright header.  Please update the
+copyright year for files you are modifying.
+
+As a heads up, we'll be running your PR through the following gauntlet:
+- warnings turned to compile errors
+- `cargo test`
+- `rustfmt`
+- `clippy`
+- `rustdoc`
+
+Check out our [CI][travis] for more information.
+
+## Releasing
+
+When we're ready to release, a project owner should do the following
+- Determine what the next version is, according to semver
+- Bump version in a commit
+  - Run `clog --setversion <X>.<Y>.<Z>`, touch up the log
```

Comment:
Yeah, killercup and I have had discussions on this (which his posts are what led me to clog).

I've been using clog with [cobalt](https://github.com/cobalt-org/cobalt.rs/blob/master/CHANGELOG.md) and [liquid](https://github.com/cobalt-org/liquid-rust/blob/master/CHANGELOG.md) and its been good enough for the time with minimal changes to the generated output.

My leaning is to just remove this line atm and not talk about changelogs and you can come in with a PR with a more detailed explanation of how you want to handle changelogs.  I propose this because if you aren't using clog, it seems like we'd want to document steps in the PR section and here and that way you can choose how you'd like.